.ci/magician/cmd/generate_comment.go

```diff
@@ -76,7 +77,7 @@ var generateCommentCmd = &cobra.Command{
 			fmt.Println("Error creating a runner: ", err)
 			os.Exit(1)
 		}
-		ctlr := source.NewController(filepath.Join("workspace", "go"), "modular-magician", env["GITHUB_TOKEN"], rnr)
+		ctlr := source.NewController(filepath.Join("workspace", "go"), "modular-magician", env["GITHUB_TOKEN_DOWNSTREAMS"], rnr)
```

Comment:
It looks like this probably needs both tokens passed in - which means refactoring the controller to support both tokens (and support only getting one or the other), or refactoring the calling code to make multiple controllers and decide which to use based on the task. I'd lean towards the second option probably.

Reply:
I don't see where the controller needs the magic modules token.

Reply:
you're right, it doesn't. I was misunderstanding what the ctlr was used for.